site/_posts/2017-03-20-yet-another-move-by-euwe.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Last Week in ManageIQ: Yet Another move by Euwe"
+author: Libor Pichler
+date: 2017-03-20 13:47:49 +01:00
+comments: true
+published: true
+tags: LWIMIQ
+---
+Hello World!
+
+My name is [Libor](https://github.com/lpichler "Can you guess what word is highlighted in my avatar?") and I am part of Platform Team in Brno.
+
+It's that time again. Time to recap of last week in ManageIQ!
+
+## Featured
+This week we were busy with moving last features to Euwe for upcoming 5.7.2 release!
+
+Let's start typically - with some numbers:
+- [88 PRs][manageiq PRs merged] merged in [ManageIQ][manageiq] core repo.
```

Comment:
Do you want to point to the [ManageIQ](https://github.com/manageiq/manageiq) repository page or the ManageIQ organization page?